### ISSUE TITLE
IllegalHeaderNameException in media uploader

### ISSUE THREAD

**Issue by 33kk** [flex3r/DankChat#291]
![Screenshot_20220723-130158_DankChat](https://user-images.githubusercontent.com/23123008/180603455-c97867e3-1c4b-42ed-bc0b-ed5bbded486d.png)

The same value works in chatterino.

```
Accept: application/vnd.chibisafe.json ; Token: feelsdankman
```

**Comment by flex3r** [comment]
Can you try without the spaces between the `;` separator? I might not trim the fields at the moment

**Comment by 33kk** [comment]
Still doesn't work, now with a generic "Upload failed error"

**Comment by flex3r** [comment]
Fixed in 1ceddbfb0cefadad0c687508620c14af9bd0d6b5
